### ISSUE TITLE
export configs

### ISSUE THREAD

**Issue by Tobi-De** [falcopackages/fujin#5]
`config expand` command that will generate systemd file for all processes in the a deploy (or maybe .fujin) folder in the current directory so that the user can customize each systemd service file by hand, also generate web service config file.
When these files are present locally they will be used instead of the auto generated ones

**Comment by Tobi-De** [comment]
```shell
fujin app export-config # will export the systemd configs
fujin proxy export-config # will export the proxy (nginx or caddy config)
```

**Comment by Tobi-De** [comment]
Implemented in 84ee3afe1b23878a72526ebe13f4dcaf72adebc8 and e9cbdc80e474164255bfa489634aedfb8bb0d3a9
